client-core/src/service/wallet_service.rs

```diff
@@ -101,11 +103,15 @@ pub struct WalletInfo {
     /// wallet meta data
     #[serde(deserialize_with = "deserde_from_str", serialize_with = "serde_to_str")]
     pub wallet: Wallet,
-    /// private key of the wallet
+    /// private key of view key pair
     #[serde(deserialize_with = "deserde_from_str", serialize_with = "serde_to_str")]
     pub private_key: PrivateKey,
     /// passphrase used when import wallet
     pub passphrase: Option<SecUtf8>,
+    ///public_key -> encoded private_key pairs, private key is None for hardware wallet
+    pub key_pairs: BTreeMap<PublicKey, Option<Vec<u8>>>,
+    /// hex encoded root_hash -> parity_scale_codec encoded multisig_address pairs
+    pub multisig_address_pair: BTreeMap<String, Vec<u8>>,
```

Comment:
What's the purpose of new fields: "key_pairs", "multisig_address_pair"

Reply:
* key-pairs: before the key-pairs stored in `Wallet`, and @leejw51 moved them out of it and restored as key-value directly in DB, so we should add it.

* multisig_address_pair: same as key-pair, or the transfer addresses and multisig address will be lost after wallet imported.